### PR TITLE
Feature/go client array support

### PR DESCRIPTION
**Pull request by lukasmittag**

### CODE REVIEW

File: kuksa_go_client/main.go [hunk 1]
```diff
@@ -51,55 +52,110 @@ if *protocol == "ws" {
...
+
+	values, err = backend.GetValueFromKuksaVal("Vehicle.OBD.DTCList", "value")
+	if err != nil {
+		log.Printf("Get Value Error: %v", err)
+	} else {
+		for _, value := range values {
+			if *protocol == "grpc"{
+				log.Println("Vehicle.OBD.DTCList: " + value.(*v1.DataEntry).String())	
+			}else{
+				log.Println("Vehicle.ADAS.ABS.IsEnabled: " + value.(string))
```
Comment (SebastianSchildt) [inline kuksa_go_client/main.go:103]: Should also be DTCList in print?

File: kuksa_go_client/main.go [hunk 2]
```diff
@@ -51,55 +52,110 @@ if *protocol == "ws" {
...
```
Comment (SebastianSchildt) [inline kuksa_go_client/main.go:124]: DTCList in print

File: kuksa_go_client/main.go [hunk 3]
```diff
@@ -51,55 +52,110 @@ if *protocol == "ws" {
...
 
 // Get MetaData of Vehicle.Speed
 value, err := backend.GetMetadataFromKuksaVal("Vehicle.Speed")
 if err == nil {
 	for _, val := range value {
-		log.Printf("Vehicle.Speed Metadata: " + val)
+		if *protocol == "grpc"{
+			log.Println("Vehicle.Speed Metadata: " + val.(*v1.DataEntry).String())
+		}else{
+			log.Println("Vehicle.ADAS.ABS.IsEnabled: " + val.(string))
```
Comment (SebastianSchildt) [inline kuksa_go_client/main.go:157]: Needs to be Speed in println

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
It seems the paths in some println outputs are incorrect

**Comment by erikbosch** [comment]
Could you do a quick check if your comments have been addressed @SebastianSchildt ?

**Pull request merged**